README.md

```diff
@@ -0,0 +1,27 @@
+<!-- PROJECT DESCRIPTION -->
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> The Hello Microverse Project
+## Built With
+
+- Webhint
+- Styleint
+
+## Authors
+
+👤 **Clinton Younge**
+
+- GitHub: [@ClintonYounge](https://github.com/ClintonYounge)
+- Linkedin: [@clinton-younge](https://www.linkedin.com/in/clinton-younge-83386a25a/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] Kindly make sure to update your README by using the [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and make sure to customize it according to your project. :wink:

Reply:
Hey @levy002 , thanks for your time. I updated the README.md to the best of my knowledge in using templates. Mind giving some feedback on the changes I made?